Asterix/wfsc/estimator.py

```diff
@@ -209,7 +209,19 @@ def __init__(self,
                                  self.dimEstim] = self.PWMatrix[k][:, 0, i].flatten()
                         vectorPW[1, i * self.dimEstim * self.dimEstim:(i + 1) * self.dimEstim *
                                  self.dimEstim] = self.PWMatrix[k][:, 1, i].flatten()
-                    namepwmatrix = '_PW_' + testbed.name_DM_to_probe_in_PW + '_wl' + str(int(wave_k * 1e9))
+
+                    # Because the exact laser WLs can change a bit quite often and labview reads a given
+                    # fits name, we name the PW matrix laser1, 2, 3 for a range of WL.
+
+                    string_laser = ''
+                    if 625 < wave_k * 1e9 < 645:
+                        string_laser = "_laser1"  # ~635nm laser source
+                    if 695 < wave_k * 1e9 < 715:
+                        string_laser = "_laser2"  # ~705nm laser source
+                    if 775 < wave_k * 1e9 < 795:
```

Comment:
What happens if I want to run this at wvln = 750 nm in simulation?

Reply:
This is not the simulatiom, this is the matrix taht are saved for the testbed in the labview directory and there are no laser at 750 on the tested. Pierre complained that because the code was saving with the actual wl in the simulation but labview was not updating matrix name. So if you change the simulation from 786 to 787, the matrix could not be read with the testbed. I just put some ranges so that it easier

Reply:
Sure, but does this mean if I run this in simulation at one of the wavelengths not covered here, and forget to set `save_for_bench=False`, that it will raise an error? It might be worth catching that error and letting the code continue.